backend/src/nodes/impl/onnx/utils.py

```diff
@@ -2,8 +2,8 @@
 
 from typing import Literal, Tuple
 
-import onnx
 import onnxruntime as ort
+from onnx.onnx_pb import ModelProto
 
 try:
     import onnxoptimizer
```

Comment:
Do we want to keep this import as optional?

Also, we should definitely keep `safely_optimize_onnx_model`. It nicely abstracts away the dependency.